tapir/shifts/templates/shifts/user_shifts_overview_tag.html

```diff
@@ -157,6 +174,11 @@
                         {% blocktranslate with used_solidarity_shifts_current_year=used_solidarity_shifts_current_year %}
                             <p>There are Solidarity Shifts available for you to use. You used {{ used_solidarity_shifts_current_year }} out of 2 Solidarity Shifts this year</p>
                         {% endblocktranslate %}
+                    {% elif used_solidarity_shifts_current_year < 2 and user.shift_user_data.get_available_solidarity_shifts and user.shift_user_data.get_account_balance <= 0 %}
+                        {% blocktranslate %}
+                            <p>You cannot receive a Solidarity Shift at the moment</p>
```

Comment:
Would it make sense to add explanation to the user why they cannot receive it?
Plus also the translation is missing.

Reply:
I also thought about that. Maybe we could use a tooltip (https://getbootstrap.com/docs/5.3/components/tooltips/#custom-tooltips) that shows some more info when a user hovers over the text?

Reply:
Good idea.